xml/System/DateTimeOffset.xml

```diff
@@ -5472,13 +5662,23 @@ Me.ToUtcDateTime().ToFileTime()
 |<xref:System.Globalization.DateTimeStyles.None>|Indicates that additional white space is not permitted in `input`. White space must appear exactly as specified in `format`. This is the default behavior.|  
 |<xref:System.Globalization.DateTimeStyles.RoundtripKind>|Has no effect, because the <xref:System.DateTimeOffset> structure does not include a `Kind` property.|  
   
-   
-  
-## Examples  
+## Examples 
+
  The following example uses the <xref:System.DateTimeOffset.TryParseExact%28System.String%2CSystem.String%2CSystem.IFormatProvider%2CSystem.Globalization.DateTimeStyles%2CSystem.DateTimeOffset%40%29> method with standard and custom format specifiers, the invariant culture, and various <xref:System.Globalization.DateTimeStyles> values to parse several date and time strings.  
   
  [!code-csharp[System.DateTimeOffset.TryParseExact#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.DateTimeOffset.TryParseExact/cs/TryParseExact.cs#1)]
  [!code-vb[System.DateTimeOffset.TryParseExact#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.DateTimeOffset.TryParseExact/vb/TryParseExact.vb#1)]  
+
+The following example uses a variety of <xref:System.Globalization.DateTimeStyles> values to parse an array of strings that are expected to conform to [ISO 8601](~/docs/standard/base-types/standard-date-and-time-format-strings.md#the-round-trip-o-o-format-specifier). As the output from the example shows, strings that are in the proper format fail to parse if:
+
+- they contain white space, and an appropriate <xref:System.Globalization.DateTimeStyles> flag (such as <xref:System.Globalization.DateTimeStyles.AllowWhiteSpace?displayProperty=nameWithType> has not been supplied in the method call.
+
+- they contain date and time elements that are out of range.
+
+Strings that do not specify a UTC offset are assumed to have the offset of the local time zone (in this case, -07:00) unless the <xref:System.Globalization.DateTimeStyles.AssumeUniversal?displayProperty=nameWithType> flag is supplied in the method call. In that case, they are assumed to be Universal Coordinated Time.
```

Comment:
Hmm. The phrase "(in this case, -7:00)" is only accurate during daylight savings time. Otherwise, it would be -8:00 UTC. These samples aren't interactive, so the behavior won't change from the comments, but it might be worth mentioning.